content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -124,38 +132,56 @@ based on formulas.
 </applications/sales/subscriptions/products>`. Be sure to check out the Odoo *Subscriptions*
-:doc:`documentation </applications/sales/subscriptions>`.
+:doc:`documentation </applications/sales/subscriptions>` to learn more.
 
-Under the :guilabel:`Time-based rules` tab, the same functionality of the :guilabel:`Price Rules`
-tab is present, with the only difference being that a repeating time period can be applied in the
-:guilabel:`Period` column.
+Under the :guilabel:`Recurring Prices` tab, the same functionality of the :guilabel:`Price Rules`
+tab is present, with the only difference being that a recurring time period can be applied in the
+:guilabel:`Recurring Plan` column.
 
-After a :guilabel:`Product` and potential :guilabel:`Variant` are selected in the
-:guilabel:`Time-based rules` tab, select the blank field in the :guilabel:`Period` column to reveal
-a drop-down menu of pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).
+Click :guilabel:`Add a price rule`, and after :guilabel:`Products`, and potential :guilabel:`Product
+Variants`, are selected, select the blank field in the :guilabel:`Recurring Plan` column to reveal a
+drop-down menu of pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).
 
-New recurrence periods can also be directly created from this column, by typing in the new name for
-the :guilabel:`Period` and then selecting :guilabel:`Create` to create the time period, which can be
-edited later. Or, select :guilabel:`Create and edit...` to reveal a pop-up form, in which the new
-recurrence period can be directly configured.
+New recurrence periods can also be directly created from this column. To do so, type in the name for
+the new :guilabel:`Recurring Plan`, then select :guilabel:`Create` from the resulting drop-down menu
+to create the time period, which can be edited later.
+
+Or, select :guilabel:`Create and edit...` to reveal a :guilabel:`Create Recurring Plan` pop-up form.
 
 .. image:: pricing/time-period-popup.png
    :align: center
    :alt: Custom time period pop-up form in Odoo Sales.
 
-From this :guilabel:`Create Period` pop-up form, add a :guilabel:`Name`, :guilabel:`Duration`, then
-a :guilabel:`Unit` (e.g. `Days`, `Weeks`, etc.). When finished, click :guilabel:`Save and Close`.
+From this pop-up form, the new recurrence period can be directly configured, with specific
+:guilabel:`Details`, :guilabel:`Self-Service`, and :guilabel:`Pricing` options. Then, when the
+configurations are complete, click the :guilabel:`Save & Close` button.
 
-Lastly, add the desired price for this time-based rule in the :guilabel:`Price` column.
+Lastly, add the desired price for this recurring price rule in the :guilabel:`Recurring Price`
+column.
 
 .. seealso::
    :doc:`/applications/sales/subscriptions`
 
+Rental rules tab
+----------------
+
+Under the :guilabel:`Rental rules` tab, specific price rules can be configured for various rental
```

Comment:
Should "rules" be capitalized?

Reply:
In this instance, no. That's because we are supposed to represent the look (and capitalization) of tabs, fields, etc. as they appear on the UI. And, in this case, the "rules" of "Rental rules" is **not** capitalized. But great attention to detail. 👍 

![Screenshot 2024-07-17 142405](https://github.com/user-attachments/assets/3832e731-5cd6-4365-95b7-5dd588e2524b)